src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -3381,6 +3381,26 @@ public void testPidsLimit() throws Exception {
 
     assertThat(info.hostConfig().pidsLimit(), is(100));
   }
+  
+  @Test
+  public void testReadonlyRootfs() throws Exception {
+    requireDockerApiVersionAtLeast("1.22", "ReadonlyRootfs");
```

Comment:
Hasn't this been available since 1.18? https://docs.docker.com/engine/reference/api/docker_remote_api_v1.18/